tools/rapids-is-release-build

```diff
@@ -8,9 +8,10 @@
 set -e
 export RAPIDS_SCRIPT_NAME="rapids-is-release-build"
 
-if [[ "${GITHUB_REF}" =~ ^refs/tags/v[0-9]{1,2}.[0-9]{2}.[0-9]{2}$ ]]; then
+if [[ "${GITHUB_REF}" == refs/tags/v* ]]; then
```

Comment:
Need to exclude alpha tags (like https://github.com/rapidsai/cudf/releases/tag/v24.08.00a).

Maybe keep the regex but allow for any number of digits?
```suggestion
if [[ "${GITHUB_REF}" =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Reply:
@raydouglass this should work no?

`if [[ "${GITHUB_REF}" == refs/tags/v* && "${GITHUB_REF}" != *a ]]; then`

Reply:
Hmm, thinking about it, that sure would not work for tags like `v24.08.00b` or `v24.08.00c` etc

Reply:
Yeah, a `b` suffix was used in the distant past (https://github.com/rapidsai/cudf/tree/v0.12.0b) and it's unlikely we would use it again, but I think this match should be a strict as possible. 

We can add exceptions if needed later.